docs/GettingStarted.md

```diff
@@ -34,6 +34,8 @@ This will build the following binaries:
                  utility functionality.
 * end_to_end.exe: A collection of tests using the Catch framework.  These tests are also run as part
                   of the Github CI/CD so should always pass.
+* ebpf_client.exe: A collection of program verification tests that exercises the RPC channel from client to ebpfsvc.
```

Comment:
```suggestion
* ebpf_client.exe: A collection of program verification tests that exercises the RPC channel from ebpfapi to ebpfsvc.
```

Reply:
ebpf_client.exe makes RPC call for program verification. The API which ebpfapi.dll exposes for program verification does not make RPC call and does verification locally by making IOCTL calls to EC. So I think it should be client instead of ebpfapi?